pypi_mirror.py

```diff
@@ -148,15 +155,15 @@ def main():
     needs_initialization = (
         not os.path.exists(local_index_path)
         or not os.path.exists(last_serial_path)
-        or not are_all_pkgs_downloaded(local_index_path)
+        # or not are_all_pkgs_downloaded(local_index_path)
```

Comment:
Nitpick: let's remove this line, please. We can always restore it from git if need to 🙂